tsl/test/expected/continuous_aggs_refresh.out

```diff
@@ -176,7 +176,15 @@ ERROR:  invalid input syntax for type timestamp with time zone: "xyz"
 CALL refresh_continuous_aggregate('daily_temp', '2020-05-03', 'xyz');
 ERROR:  invalid input syntax for type timestamp with time zone: "xyz"
 CALL refresh_continuous_aggregate('daily_temp', '2020-05-03', '2020-05-01');
-ERROR:  invalid refresh window
+ERROR:  refresh window too small
+-- Refresh window less than one bucket
+CALL refresh_continuous_aggregate('daily_temp', '2020-05-01 05:00', '2020-05-01 06:00');
+ERROR:  refresh window too small
+-- Refresh window bigger than one bucket, but not covering a full bucket
+-- Refresh window:    [----------)
+-- Buckets:          [------|------]
```

Comment:
For consistency with code, shouldn't this say, "Refresh window bigger than one bucket, but a minimum of two buckets is required"??

Reply:
same in the `...refresh.sql` below.

Reply:
This one is a bit tricky, and certainly something to discuss. For the manual refresh using  `CALL refresh_continuous_aggregate`  it is perfectly viable to refresh one bucket. For instance, if you have a 1 day bucket cagg and run:

```
CALL refresh_continuous_aggregate('temp_1day', '2021-02-01', '2021-02-02');
```
You'd expect that to work, right? However, there's a caveat since the refresh window must perfectly align with bucket boundaries, which it would if you are in bucketing time zone (UTC). The issue is that if your own time zone is not UTC, then you'd have to specifically specify UTC:


```
CALL refresh_continuous_aggregate('temp_1day', '2021-02-01 00:00 UTC', '2021-02-02 00:00 UTC');
```

This is because `time_bucket` doesn't support the time zone arg, like `date_trunc` does. (Although we have an offset parameter, but it currenlty doesn't work with caggs I believe).

I think for policies, it makes sense to enforce two buckets since there is no real way to control when policies run so it would be impossible to align the refresh window with bucket boundaries.

I added additional test cases and tweaked errors to make these issues more clear.